parts/k8s/kubernetesmastercustomdata.yml

```diff
@@ -86,6 +86,26 @@ write_files:
       name: localclustercontext
     current-context: localclustercontext
 
+{{if EnableDataEncryptionAtRest}}
+- path: "/etc/kubernetes/encryption-config.yaml"
+  permissions: "0600"
+  owner: "root"
+  content: |
+    apiVersion: v1
+    kind: Config
+    kind: EncryptionConfig
```

Comment:
2 kind fields?

Reply:
I think @mrosterm is right. @pidah could you submit a follow-up PR to remove the `kind: Config` line? Based on the docs you provided that's the resource type we want.

Thanks both!

Reply:
@jackfrancis sorry just saw this, will raise a fix PR.